.github/workflows/conformance.yml

```diff
@@ -0,0 +1,23 @@
+name: Conformance Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  conformance:
+    permissions:
+      # Needed to access the workflow's OIDC identity.
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@d171c3b028d844f2bf14e9fdec0c58114451e4bf
+      - uses: actions/setup-python@7f80679172b057fc5e90d70d197929d454754a5a
+      - name: install sigstore-python
+        run: python -m pip install .
+      - uses: trailofbits/sigstore-conformance@v0.0.1
```

Comment:
nit: could use hash pinning like the other actions do -- dependabot* should still follow your latest release.

*) just realized dependabot does not seem to be set up for GH actions currently: will file an issue

Reply:
[6ca2ed7](https://github.com/sigstore/sigstore-python/pull/298/commits/6ca2ed7d0212af7dddfefe92d68cf88445bcf9c9)